paf-mvp-frontend/rollup.config.js

```diff
@@ -107,8 +108,19 @@ export default [
           ]
         } else { // list of plugins for development
           return [
+            copy({ // copy files
+              targets: [
+                {
+                  src: './assets/*',
+                  dest: 'dist',
+                },
+              ],
+            }),
             serve({ // dev server
               contentBase: '',
+              headers: {
+                'Access-Control-Allow-Origin': '*'
```

Comment:
Is it the same as the production configuration ?
If not, I am wondering if things can work locally but break on production.

Reply:
dev server is for local development only. I added this header for fonts files on the prod server